.storybook/utils/styled-system-props.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import PropTypes from "prop-types";
 import { ArgsTable } from "@storybook/components";
```

Comment:
This `ArgsTable` is actually different to the one imported from `@storybook/addon-docs` for anyone looking 😅